ui/modules/things/attributes.ts

```diff
@@ -105,10 +110,10 @@ function refreshAttribute(thing, attributePath = null) {
 
   if (thing) {
     dom.crudAttribute.idValue = attributePath;
-    dom.inputAttributeValue.value = attributeToString(thing.attributes[attributePath]);
```

Comment:
`attributeToString` is no longer called. I have seen that simple strings are displayed with double quotes and same for numbers (see e.g. a coffeebrewer Thing with attribute `42` or `Speaking coffee machine`). Can we achieve that you can still edit simple numbers or simple strings?

Reply:
I wouldn't do that - the ACE editor is in "JSON mode" - if you put in a string without the quotes, this is no longer valid JSON (and the ACE editor will complain).
Numbers are displayed as numbers without quote (if they are saved as a JSON number in the attribute).

In JSON, `"42"` is something completely different to `42` - I think before you could not even see (and make) that difference.

The attributes table still shows the `attributeToString` representation - which is great to get a quick overview.

It is much more consistent with all other editors on the UI as well.

Reply:
Fine for me. I just checked: The sample "coffeebrewer" sample really has a `"42"` as string and not as an integer. And you still can set an integer attribute with the json editor. So that confused me. So you just need the quotes for "string attributes". All good. Thank you.